packages/kbn-securitysolution-utils/src/path_validations/index.ts

```diff
@@ -22,6 +22,21 @@ export const enum ConditionEntryField {
   SIGNER = 'process.Ext.code_signature',
 }
 
+export const enum EntryFieldType {
+  HASH = '.hash.',
+  PATH = '.executable.caseless',
+  SIGNER = '.Ext.code_signature',
+}
```

Comment:
Do we need to add these new `file...` fields to  [`common/endpoint/exceptions/exceptionable_endpoint_event_fields.ts`](https://github.com/joeypoon/kibana/blob/e6f225bc56890c3bfd4992549c09f92a638ffb5c/x-pack/plugins/security_solution/common/endpoint/exceptions/exceptionable_endpoint_event_fields.ts#L93)

Reply:
Ah, great question. I just checked now and they're already included. Thanks for reminder to check.

Reply:
We're missing `file.executable.caseless` from that list 😅